### DISCUSSION TITLE
how to update selectedRowKeys when i use preserveSelectedRowKeys as true.

### DISCUSSION THREAD

**Discussion by bachtdx** [ant-design/ant-design#31256] [Q&A]
- [x] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### Reproduction link
[https://ant.design/components/table/#API part: rowSelection: preserveSelectedRowKeys](https://ant.design/components/table/#API part: rowSelection: preserveSelectedRowKeys)

### Steps to reproduce
i can't update selectedRowKeys when i use preserveSelectedRowKeys is true

### What is expected?
i can update selectedRowKeys when i use preserveSelectedRowKeys is true

### What is actually happening?
 i can't update selectedRowKeys when preserveSelectedRowKeys is true

| Environment | Info |
|---|---|
| antd | 4.16.6 |
| React | react js |
| System | windown 10 |
| Browser | chrome  91.0.4472.114  |

**Comment by monica-boga** [comment]
is there any solution to this issue? I have the same issue and cannot find the solution?
